spa_ui/self_service/client/app/states/requests/list/list.state.js

```diff
@@ -17,13 +17,19 @@
         controller: StateController,
         controllerAs: 'vm',
         title: 'Request List',
+        params: {apiResponse: null},
         resolve: {
+          apiResponse: resolveApiResponse,
           requests: resolveRequests
         }
       }
     };
   }
 
+  function resolveApiResponse($stateParams) {
```

Comment:
I had thought that `params` are automatically available to the controller and you don't need to resolve them. Are you not seeing `apiReponse` in the controller without this resolve?